src/main/resources/data/po-lines/312325-1_fully_received_electronic_method-purchase_at_vendor_system.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "15fb695a-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "edition": "First edition",
+  "checkinItems": false,
+  "instanceId": "85122745-9347-46a6-956a-4cd5e6d6ca25",
```

Comment:
remove InstanceId, create inventory is false, (or vice versa)

Reply:
Changed

Reply:
it's still there...  Looks like you removed `agreementId`

Reply:
agreementId re-added, instanceId removed.